src/libraries/System.Private.CoreLib/src/System/Threading/NativeRuntimeEventSource.PortableThreadPool.cs

```diff
@@ -65,6 +65,27 @@ public enum ThreadAdjustmentReasonMap : uint
             ThreadTimedOut
         }
 
+#if MONO
```

Comment:
Our coding style has strong prefers separate files over ifdefs. Can this be a separate file insted?

I think the implementation that uses `WriteEventCore` should be under src/libraries/System.Private.CoreLib since it is runtime-neutral, and the implementation that uses QCalls should be in a separate file under src/coreclr/System.Private.CoreLib` since it is very CoreCLR specific.